### ISSUE TITLE
Excel Function Wizard not working if UDF arguments names are long

### ISSUE THREAD

**Issue by mattpogo** [xlwings/xlwings#1489]
#### Describe your issue (incl. Traceback!)

Hi!
I noticed the Function Wizard in Excel does not seem to work if 'the number of function arguments' + 'number of total characters in argument names' exceeds the value of 254.

For example, the wizard (sorry, German Excel) does not show the correct number of arguments nor the argument names when using the code snippet below the picture.  

![image](https://user-images.githubusercontent.com/37383481/104612497-8e6c4a00-5686-11eb-86e3-2cbedfe0d6b2.png)

Rename the last argument to g234567 and everything works.

Is that a limitation of Excel or something that can be fixed?

```python
@xw.func(category="Test")
@xw.arg('a234567890123456789012345678901234567890', doc="Some help.")
@xw.arg('b234567890123456789012345678901234567890', doc="Some help.")
@xw.arg('c234567890123456789012345678901234567890', doc="Some help.")
@xw.arg('d234567890123456789012345678901234567890', doc="Some help.")
@xw.arg('e234567890123456789012345678901234567890', numbers=int, doc="Some help.")
@xw.arg('f234567890123456789012345678901234567890', numbers=int, doc="Some help.")
@xw.arg('g2345678', numbers=int, doc="Some help.")
def fal_a(a234567890123456789012345678901234567890,
          b234567890123456789012345678901234567890,
          c234567890123456789012345678901234567890,
          d234567890123456789012345678901234567890,
          e234567890123456789012345678901234567890,
          f234567890123456789012345678901234567890,
          g2345678):
    """Returns something."""
    pass
```

**Comment by fzumstein** [comment]
It's an Excel limitation but we should be able to handle it better (for example the function's doc string is already automatically cropped to the max length.).

**Comment by mattpogo** [comment]
Thanks for the quick answer!
I haven't found anything in the documentation regarding this limitation. If not documented yet, that would be a helpful. And if it could be handled more gracefully, even better.

**Comment by Colin-b** [comment]
Yes the fact is that those limitation are not documented on MS side, just so you know, another one I encountered is that you can only list a certain amount of functions per category, they are still accessible, but not listed in the UI.
